classes.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class DbConnect
```

Comment:
the name of the class should be like the name of a thing - e.g. DbConnection or some other noun